agents/chatembed/main.py

```diff
@@ -0,0 +1,19 @@
+import fixieai
+
+BASE_PROMPT = (
+    """Hi, I'm a helpful chatbot who can answer questions from an uploaded file."""
+)
+
+FEW_SHOTS = """
+Q: When did the thing happen? #text1
+Ask Func[fixie_query_embed]: When did the thing happen? #text1
+Func[fixie_query_embed] says: It happened on March 1, 2023
+A: The thing happened on March 1, 2023.
+
+Q: What was the thing?
+Ask Func[fixie_query_embed]: What was the thing? #text1
```

Comment:
This `#text1` does not exist in query and so cannot be used as an argument to fixie_query_embed. It's a good fewshot validation check we should consider adding (cc @petersalas).

Reply:
Yeah, I am reworking this to store the embed in user storage.

Reply:
You could also make this agent conversational and stick the 2 fewshots together to teach the model it should bring the embedding from history.

Reply:
That's what I ended up doing. I also leaned on base prompt to do more of the lifting here, as otherwise it wouldn't add the embed reference for any sequence longer than the longest few-shot.